lua/terrortown/menus/gamemode/equipment.lua

```diff
@@ -33,9 +33,15 @@ function CLGAMEMODEMENU:InitializeVirtualMenus()
 
 		counter = counter + 1
 
+		local isItem = items.IsItem(equipment)
+
+		if not isItem then
+			equipment = weapons.Get(WEPS.GetClass(equipment))
+		end
```

Comment:
I forgot a comment here:
```suggestion
		-- Get inheritable version for weapons
		if not isItem then
			equipment = weapons.Get(WEPS.GetClass(equipment))
		end
```